packages/cli/src/bin/pipcook-plugin.ts

```diff
@@ -46,6 +52,49 @@ async function list(opts: any): Promise<void> {
   }
 }
 
+async function upload(localPluginsPath: string, opts: any): Promise<void> {
```

Comment:
Do we support uploading multiple plugins as the first argument name is localPlugin"s"Path.